tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/contentassist/CompletionWithASITest.xtend

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) 2020 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * Contributors:
+ * NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ide.tests.contentassist;
+
+import java.util.List
+import org.eclipse.n4js.ide.tests.server.AbstractCompletionTest
+import org.junit.Test
+import org.junit.Ignore
+
+/**
+ * Code completion tests for scenarios that also might add an import statement 
```

Comment:
Could you add the term "automatic semicolon insertion" somewhere here because "ASI" is not that obvious, especially if I forget its meaning (which might happen soon).